software/control/src/AtlasFallDetector.cpp

```diff
@@ -91,8 +93,9 @@ void AtlasFallDetector::handleRobotState(const lcm::ReceiveBuffer* rbuf,
   // std::cout << " active: " << controller_is_active << " user: " << atlas_is_in_user << " contact valid: " << foot_contact_valid;
   if (this->controller_is_active && this->atlas_is_in_user && this->foot_contact_valid) {
     this->state_driver->decode(msg, &(this->robot_state));
-    KinematicsCache<double> cache = this->model->doKinematics( robot_state.q, robot_state.qd);
-    kinematics_cache = std::shared_ptr< KinematicsCache<double> >(&cache);
```

Comment:
ah i think this explains a segfault i was seeing in the fall detector.  I think it's been this way since October and but hasn't seen much use since then.